reportsrv/reportsrv.go

```diff
@@ -192,6 +192,7 @@ func (s *Server) doAuthzAndAuditLog(a *pacta.Analysis, aa *pacta.AnalysisArtifac
 	actorID, _ := session.UserIDFromContext(ctx)
 	if actorID == "" {
 		actorID = unauthenticatedUserID
+		actorOwner = &pacta.Owner{ID: unauthenticatedUserID}
```

Comment:
I missed this before, but why set the actor to some invalid sentinel versus just leaving it blank?

Reply:
Good question: when I was thinking about the API here I realized that empty values for `actor` or `actorOwner` would be suspicious for someone on the outside to understand. Since Audit Logs are intended for external customers, saying "oh if it's unset it's just anonymous" engenders less trust than explicitly saying so in the audit log record itself. It also distinguishes between a case of missing data and a case of an anonymous user effectively.